actors/abi/bigint/bigint.go

```diff
@@ -0,0 +1,250 @@
+package big
```

Comment:
Package name should match the path; rename the directory containing these files.